inc/saiswitch.h

```diff
@@ -2530,6 +2530,30 @@ typedef enum _sai_switch_attr_t
      * @flags READ_ONLY
      */
     SAI_SWITCH_ATTR_AVAILABLE_DOUBLE_NAPT_ENTRY,
+    
+    /**
+     * @brief Parallel processing attributes to lock shared resources to achieve thread/process synchronization
+     *
+     *  The attributes data type is void pointer and the format of the function pointers are as follows:
+     *  int brcm_mutex_lock(uint64_t platform_context)
```

Comment:
are those callbacks ?, why they are broadcom specific ?
if so they would need to be specifically added as SAI_SWITCH_ATTR_REGISTER_WRITE, and parse.pl needs to be updated

Reply:
I will take out these two attributes and will add as separate PR